website/src/models/user.js

```diff
@@ -390,6 +390,15 @@ var UserSchema = new Schema({
       hatchPet: {type: Boolean, 'default': false},
       raisePet: {type: Boolean, 'default': false},
       streak: {type: Boolean, 'default': false}
+    },
+    improvementCategories: {
```

Comment:
I think that PUT route isn't that evil. If we start creating a route for all the boolean/string/number flags we'll never end